src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html

```diff
@@ -1,10 +1,11 @@
-<div class="row justify-content-center">
+<div class="row justify-content-center" >
     <div class="col-8">
         <form name="editForm" role="form" novalidate (ngSubmit)="save()" [formGroup]="editForm">
             <h2 class="list-text-color-bold" id="custom-therapeutic-regime-heading" data-cy="TherapeuticRegimeCreateUpdateHeading" jhiTranslate="supportivecareApp.therapeuticRegime.home.createOrEditLabel">Create or edit a Therapeutic Regime</h2>
             <div class="bg-light" >
                 <h5 class="list-text-color-bold col-lg-12 mt-4 mb-4"  jhiTranslate="supportivecareApp.therapeuticRegime.generalInfo"> >General information</h5>
                 <custom-alert-error></custom-alert-error>
+
```

Comment:
Is this change needed?

Reply:
This change does not affect anything, might be to the fact that we were experimenting and forgot a blank space